cmake/init_local_projects.cmake

```diff
@@ -0,0 +1,18 @@
+if(NOT INSIDE_DOCKER)
+    if("${USE_IDE}" STREQUAL "eclipse")
+        message("Generating project files for Eclipse IDE...")
+        message(WARNING "Eclipse IDE currently not supported!")
+        # TODO: Debug Configuration
+    elseif("${USE_IDE}" STREQUAL "vscode")
```

Comment:
Here, placeholders in template files (`*.in`) located at `config/vscode/` are filled with CMake variables and the result is copied to the `.vscode` directory.